src/components/icon/GetIcon.tsx

```diff
@@ -24,7 +24,7 @@ const GetIcon: FC<GetIconProps> = ({
       sx={{
         svg: {
           width: prop(size, theme.iconSizes),
-          height: 'auto',
+          height: prop(size, theme.iconSizes),
```

Comment:
this was actually causing some problems with height sometimes being 0 and svg not displayed

Reply:
all our icons are 24x24 anyways, so square is perfectly good imo